frontend/app/root.tsx

```diff
@@ -91,16 +97,22 @@ export default function App() {
           <link key={hrefLang} rel="alternate" hrefLang={hrefLang} href={href} />
         ))}
         <Links />
+        {env.ADOBE_ANALYTICS_SRC && env.ADOBE_ANALYTICS_JQUERY_SRC && (
+          <>
+            <script src={env.ADOBE_ANALYTICS_JQUERY_SRC} />
+            <script src={env.ADOBE_ANALYTICS_SRC} />
+          </>
+        )}
       </head>
       <body vocab="http://schema.org/" typeof="WebPage">
         <Suspense>
           <Outlet />
           <Toaster toast={toast} />
         </Suspense>
-        <ScrollRestoration nonce={nonce} />
-        <Scripts nonce={nonce} />
-        <ClientEnv env={env} nonce={nonce} />
-        <LiveReload nonce={nonce} />
+        <ScrollRestoration />
+        <Scripts />
+        <ClientEnv env={env} />
+        <LiveReload />
```

Comment:
**Nitpick:** you could have left the nonce attribute here and just removed it from the CSP headers (with a `XXX` or `TODO` flag).. that way, if we ever want to re-add it back in the future, it will be easier.

Reply:
Reverted.